### ISSUE TITLE
Root namespace variable replacement

### ISSUE THREAD

**Issue by fbjerggaard** [threenine/api-template#9]
When setting the root namespace with the following command:
`dotnet new apisolution -n MyTestAPI --Root MyCompany -o .\mytestapi\`

The `Directory.Build.props` file will look like the following:

```
<Project>
    <PropertyGroup>
        <TargetFramework>net6</TargetFramework>
        <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
        <IsPackable>false</IsPackable>
        <ImplicitUsings>true</ImplicitUsings>
        <ImplicitUsings>enable</ImplicitUsings>
        <Nullable>disable</Nullable>
        <MyCompanyNamespace>MyCompany</MyCompanyNamespace>
    </PropertyGroup>
</Project>
```

Notice that `RootNamespace` has been incorrectly set to `MyCompanyNamespace`

**Comment by garywoodfine** [comment]
I will resolve this shortly!

Thanks for the feedback

**Comment by garywoodfine** [comment]
Updated the package with a fix for the namespace issue in version 3.2.62
